WorkingInR/0_Preamble.Rmd

```diff
@@ -0,0 +1,74 @@
+---
+title: "Preamble"
+output:
+  bookdown::html_document2:
+  highlight: tango
+toc: true
+toc_float: true
+css: ../css/style-chapters.css
+---
+  
+``` {r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning = FALSE)
+```
+
+
+# Preamble
+
+## About this website
+
+These pages are the result of a collaborative effort between bioCEED, Centre for Excellence in Biology Education, the Department of Biological Sciences at the University of Bergen, and the Department of Biology and the Department of Mathematical Sciences at the Norwegian University of Science and Technology.
+
+## Who is it for?
+
+
+
+## Prerequisites
+
++ functional computer and access to the internet (sais in a humoristic)
++ stats 101 ?
+
+
+## Structure of the webpages
+
+### Before we start
+
+At the top of each page, you will find the `Before we start` box which tells you what you must already know or be familiar with before you read further. 
+
+### Body
+
+The body of the page introduces the topic(s) along with relevant examples.
+
+### Interactive apps 
```

Comment:
It's interactive Learner sections... not sure what to call it.

Reply:
What about "Interactive Learnr tutorials"